improver/lapse_rate.py

```diff
@@ -326,12 +327,107 @@ def _create_heightdiff_mask(self, all_orog_subsections):
 
         height_diff = np.subtract(self.all_orog_subsections, central_points)
         height_diff = np.absolute(height_diff)
-
-        height_diff_mask = np.where(height_diff >= self.max_height_diff, True,
+        height_diff_mask = np.where(height_diff >=
+                                    self.max_height_diff, True,
                                     False)
 
         return height_diff_mask
 
+    def _rolling_window(self, A, temp_shape, shape):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)